lib/executor/ramping_vus_test.go

```diff
@@ -36,6 +37,26 @@ func TestRampingVUsConfigValidation(t *testing.T) {
 	errs = c.Validate()
 	require.NotEmpty(t, errs)
 	assert.Contains(t, errs[0].Error(), "greater than 0")
+
+	c.StartVUs = null.IntFrom(100000001)
```

Comment:
I find that this test already fulfills the job of multiple test cases by itself. Changing that is probably out of the scope of this specific PR, but would you mind revamping your addition to the test in a way that it acts as a dedicated test instead?

Something along the lines of

```golang
	t.Run("validate large VU shifts should fail", func(t *testing.T) {
	    t.Parallel()
	    
	    // ... your test code
	})
```